users/postgres/init.go

```diff
@@ -78,6 +78,13 @@ func migrateDB(db *sqlx.DB) error {
 					`ALTER TABLE IF EXISTS users ADD PRIMARY KEY (id)`,
 				},
 			},
+			{
+				Id: "users_5",
+				Up: []string{
+					`ALTER TABLE IF EXISTS users ADD COLUMN IF NOT EXISTS
+					status VARCHAR(64) NOT NULL DEFAULT 'enabled'`,
```

Comment:
Consider using SMALLINT and enumeration for status. While it is more cryptic than VARCHAR, it's also unlikely that we'll have any other status anytime soon. Unlike simple boolean, it also leaves us the space for adding potential new statuses without changing the schema in the future.

Reply:
`SMALLINT` and `ENUM` are both small and fast. `SMALLINT` have the advantage of changing to multiple values in the future without schema change but this needs to be defined somewhere so that someone can know the values for each int and are therefore more cryptic than `ENUM`.  `ENUM` has an additional advantage of being safe whereas in `VARCHAR` or `SMALLINT` you have to specify the check function. I propose the use of `ENUM` since I doubt that the values will change in the near future.